### DISCUSSION TITLE
Welcome | How to use Github Discussions

### DISCUSSION THREAD

**Discussion by tsliaw** [hubmapconsortium/portal-ui#2606] [General]
## 👋  Welcome 
This Github Discussions page is being used an internal tracker for user feedback, portal use cases, personas and any questions related to the portal. 
Any users are welcome to submit relevant topics for discussions, and to use the appropriate categories for submission. Categories for submission include: use cases, features, help desk and general. If you do not know which category to submit under, please use the 'General' category. 

## ⬛️ Features
Submit features that you would like to see on the portal. Be as descriptive as possible about your request.

## 🙏 Help Desk
If you are having trouble with any part of the portal, submit your feedback in this category. 

## 💡 Use Case
If you have a use case for the portal that has not already been submitted as a discussion topic, submit your use case here and use the template provided in the discussions (template pinned in discussion main page).

## 💬 General
Any topics not related to the above categories, categorize in this section.

**Comment by RedStarDreamer** [comment]
I just signed up for Github today and am a bit overwhelmed. It looks interesting though. I am currently working on the Google Data Analytics certificate program.
